src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -230,19 +223,34 @@ protected override bool Receive(object message)
         {
             switch (message)
             {
-                case KafkaConsumerActorMetadata.Internal.Commit commit when _rebalanceInProgress:
-                    _rebalanceCommitStash = _rebalanceCommitStash.Union(commit.Offsets);
-                    _rebalanceCommitSenders = _rebalanceCommitSenders.Add(Sender);
-                    return true;
-
                 case KafkaConsumerActorMetadata.Internal.Commit commit:
-                    _commitRefreshing.Add(commit.Offsets);
```

Comment:
Changed what `Commit` does, as it was basically a bulk commit-everything-now message before. Now it gives us new `TopicPartitionOffset`s to add to the `_commitMaps` - and we will commit this stuff in aggregate in the near future.